SignalR.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26228.9
```

Comment:
Wow...  VS seems to have barfed all over my SLN. Not sure if I should revert it if it's only going to do it again...